ntoskrnl/cc/mdl.c

```diff
@@ -99,12 +99,15 @@ CcMdlReadComplete (
     FastDispatch = DeviceObject->DriverObject->FastIoDispatch;
 
     /* Check if we support Fast Calls, and check this one */
-    if (FastDispatch && FastDispatch->MdlReadComplete)
-    {
-         /* Use the fast path */
+    /* Use the fast path */
+    if (FastDispatch != NULL &&
+        FastDispatch->MdlReadComplete != NULL &&
         FastDispatch->MdlReadComplete(FileObject,
```

Comment:
Please separate this into it's own ```if``` statement. Same below.

Reply:
Done.